internal/driver/fetch.go

```diff
@@ -378,14 +379,31 @@ func collectMappingSources(p *profile.Profile, source string) plugin.MappingSour
 // set to the remote source URL by collectMappingSources back to empty string.
 func unsourceMappings(p *profile.Profile) {
 	for _, m := range p.Mapping {
-		if m.BuildID == "" {
+		if m.BuildID == "" && !isWindowsAbsPath(m.File) {
 			if u, err := url.Parse(m.File); err == nil && u.IsAbs() {
 				m.File = ""
 			}
 		}
 	}
 }
 
+// isWindowsAbsPath detects whether path starts with drive letter such as "C:\".
```

Comment:
Can we just do `if m.BuildID && filepath.VolumeName(m.File) == ""`?

Reply:
Gentle ping.

Reply:
Following up on this, apologies for the delay. I am hesitant to claim pprof support of opening Windows-collected profiles on Linux. This seems exotic. Hence, I would suggest we use `filepath.VolumeName` here. Similar below in the trimming code.

Reply:
It does happen quite often for me, but very well, I'll use filepath.VolumeName.